### DISCUSSION TITLE
思源宋体 | /cn/2017/04/source-han-serif/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#912] [Comments]
https://yihui.org/cn/2017/04/source-han-serif/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
字体的止境暂时就在这里了，我估计现在这个设置可以维持两年。我之所以花时间在这里是因为这是第一款我中意的免费开源中文宋体，而且又知道了网络字体对中文也不是不可能。

中英混排右边缘的问题是挺恶心的，但中文问题在软件里的优先级通常比较低（除非是纯中文应用），因为开发者里面华人应该是少数，这些问题恶心不到开发者，就像 RStudio 编辑器或者 R 本身的中文支持问题一样，我们公司就我一个中国人，通常只能靠我来吼。

> ——原帖发布于 2017-04-21 14:42:58

**Reply by yihui** [reply]
我不知道啊，我现在不太用 Linux 了。你可以去论坛上汇报一下，但我估计修复的可能性不太大：<https://support.rstudio.com>

> ——原帖发布于 2017-08-15 14:20:25

**Reply by yihui** [reply]
土著华人应该还是只有我。有好几个台湾后裔。

> ——原帖发布于 2019-05-11 14:36:15

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
读者也许不挑脸，但脸丑非常妨碍我写东西，字体好看是我写作的首要动力。

> ——原帖发布于 2017-04-21 14:44:12

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
颁发今日最用心观众奖章一枚。

> ——原帖发布于 2017-04-26 03:03:08

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我估计是衬线字体比非衬线字体的制作难度高很多，开源社区难以承受这个工作量，Adobe 也是几年前先做完了黑体，现在才刚刚做完宋体。文泉驿也有宋体，但我不知道样式如何。

> ——原帖发布于 2017-04-27 15:41:28

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
Homebrew 千秋万载一统江湖！

> ——原帖发布于 2017-05-13 00:11:54

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
居然有差别对待……

> ——原帖发布于 2017-05-15 18:09:11

[a comment by an automated account is not shown]

**Comment by yihui** [comment]
Google Fonts 里没有思源宋体啊，而且它不支持按需加载，而且 Google 的东西在国内访问肯定是要翻墙的。我在 iPhone 上用 Safari 看我的页面可以看见思源宋体，所以我不知道所谓的 Typekit 不支持移动设备是怎么一回事。

> ——原帖发布于 2019-01-08 17:22:41

[a comment by an automated account is not shown]

**Comment by think-crow** [comment]
老歪，你好！思源宋体这篇文章的windows一词，在我这边中文环境下显示的是瘟到死系统

**Reply by yihui** [reply]
哈哈，那是我年少轻狂无知的时候喜欢嘲讽 Windows，所以每次都把它称为“瘟到死”。

诶？请问你是哪过？怎么感觉像是我的高中同学？

**Reply by think-crow** [reply]
意外意外，回复这么快，哈哈哈！在下内地隶属中原地区毕业五六年的愤青男：姓任、名豪爽！
可惜不是你高中同学。

看过你最早的文章：了解了你的别名老歪，这篇文章太个性了，简直鼻祖（歪我很喜欢-不走大道，哈哈哈）
还有你的关于-知道了不在乎称呼，那咱就称呼你老歪！

**Reply by yihui** [reply]
吓我一跳，这个外号只有我的中学同学知道，而且连他们都已经很少这样叫我了，我还以为是哪个老同学来了。

另外，这篇文章介绍的办法我已经换成了另一种更稳健的解决方案：https://github.com/yihui/yihui.org/discussions/1547

**Reply by think-crow** [reply]
哈哈哈！好滴！感谢！
